arches/app/views/auth.py

```diff
@@ -223,6 +222,10 @@ def get(self, request):
             },
         )
 
+    @method_decorator(
+        sensitive_variables(),
+        sensitive_post_parameters(),
+    )
```

Comment:
This only seems to work when the methods are within an iterator (as I see you've done elsewhere)